ReSharper.FSharp/src/FSharp.Psi.Features/src/FileTemplates/AzureFSharpScopeProvider.fs

```diff
@@ -1,5 +1,5 @@
 namespace JetBrains.ReSharper.Plugins.FSharp.Psi.Features.FileTemplates
-
+(*
```

Comment:
What happened to `InAzureFunctionsProject` and the others? cc @ForNeVeR

Reply:
@DedSec256 Could you please delete this code completely in a separate PR?

Reply:
These types were moved to Azure Toolkit plugin; feel free to remove this code from here.